module/shikshalokam/helper.js

```diff
@@ -135,4 +135,41 @@ module.exports = class shikshalokamHelper {
         })
     }
 
+     /**
+     * Organisation and root organisation data
+     * @method
+     * @name getOrganisationsAndRootOrganisations
+     * @param {String} token - token data.
+     * @param {String} userId - Logged in user id.
+     * @returns {Array} - Created for and root organisation details.
+     */
+
+    static getOrganisationsAndRootOrganisations(token,userId) {
```

Comment:
@aman-tunerlabs - Verify if org and root org is coming up correctly in v1 create, v2 create and bulk create